core/include/optee_rpc_cmd.h

```diff
@@ -148,6 +148,28 @@
  */
 #define OPTEE_RPC_CMD_BENCH_REG		20
 
+/*
+ * Issue master requests (read and write operations) to an I2C chip.
+ *
+ * [in]     value[0].a	    Transfer mode (check enum rpc_i2c_mode for modes).
+ * [in]     value[0].b	    The I2C bus (a.k.a adapter).
+ *				16 bit field.
```

Comment:
This ABI does not really care `bus`, `chip` and `flags` fit in 16bit cell that is rather an implementation choice in optee.

Suggest simply:
```
 * [in]     value[0].b	    The I2C bus (a.k.a adapter).
 * [in]     value[0].c	    The I2C chip (a.k.a address).
 * [in]     value[1].a	    Control flags (OPTEE_MSG_RPC_CMD_I2C_FLAGS_*)

Reply:
@etienne-lms @jenswi-linaro can you agree on this and let me know. my understanding was that @jenswi-linaro wanted to know what bits were being used on each field for extensibility

Reply:
Correct, better to allow just what's needed.